osbs/api.py

```diff
@@ -713,6 +713,57 @@ def get_build_logs(self, build_id, follow=False, build_json=None, wait_if_missin
 
         return logs
 
+    @staticmethod
+    def _parse_build_log_entry(entry):
+        items = entry.split()
+        if len(items) < 4:
+            logger.warning("This is not a valid build log entry: %s", entry)
+            return (None, entry)
+
+        platform = items[2]
+        if not platform.startswith("platform:"):
+            logger.warning("Line logged without using the appropriate LoggerAdapter")
+            return (None, entry)
+
+        platform = platform.split(":")[1]
```

Comment:
We won't have platform names that themselves contain ":" I'm sure, but just to avoid adding that additional constraint could we use `split(':', 1)[1]` here?